test/app/account_test.rb

```diff
@@ -0,0 +1,26 @@
+require_relative '../app_helper'
+
+class AppExercisesTest < Minitest::Test
+  include Rack::Test::Methods
+  include AppTestHelper
+  include DBCleaner
+
+  def app
+    ExercismWeb::App
+  end
+
+  attr_reader :bob, :alice
+
+  def setup
```

Comment:
@kytrinyx , I was trying to add my test users in the `def setup` (kind of like using `def before`, right?) but for some reason only the last user was being created. Do you know why that may be?